src/sdk/namespace_s3.js

```diff
@@ -513,7 +534,7 @@ class NamespaceS3 {
             obj_id: res.UploadId || etag,
             bucket: bucket,
             key: res.Key,
-            size: ranges || res.ContentLength || res.Size || 0,
+            size: res.ContentLength || ranges || res.Size || 0,
```

Comment:
Wait, is that safe??

Reply:
Ran into issue with content-length in noobaa's response header to s3 client when partNumber presents. The changes is outdated. I added content_length in the return of the function.